src/Container/ObjectRepositoryRoleProviderFactory.php

```diff
@@ -33,8 +33,10 @@
  */
 final class ObjectRepositoryRoleProviderFactory
 {
-    public function __invoke(ContainerInterface $container, $requestedName, ?array $options = []): ObjectRepositoryRoleProvider
+    public function __invoke(ContainerInterface $container): ObjectRepositoryRoleProvider
     {
+        $options = $container->get('config')['zfc_rbac']['role_provider'][ObjectRepositoryRoleProvider::class] ?? [];
```

Comment:
I am not sure about `$container->get('config')` this is ZF thing. Maybe someone can suggest something?

Reply:
how do Symfony and others pull config from their container? Looks pretty standard to me, dunno if we could generalize some this trivial...

Reply:
I think I could use `ZfcRbac\Options\ModuleOptions` here will try but anyway `ZfcRbac\Options\ModuleOptions` itself uses `$container->get('config')['zfc_rbac']` also maybe we don't need those factories? It's not that hard for the user to write themselves the way they need and want. This would also allow us to remove role_provider config option is a good thing no?